.github/workflows/diff_report.yml

```diff
@@ -15,6 +15,7 @@ name: Diff-Report
 env:
   AWS_REGION: us-east-2
   AWS_BUCKET_NAME: "checkstyle-diff-reports"
+  DEFAULT_PROJECTS_LINK: "https://raw.githubusercontent.com/checkstyle/contribution/master/checkstyle-tester/projects-to-test-on.properties"
```

Comment:
please create PR to activate more/almost-all project in this file.
Execution in github should not exceed time limit of Github action.

Reply:
We also need to send PR to update doc as it will be our primary way that we suggest users to use.

Reply:
PR for uncommented projects https://github.com/checkstyle/contribution/pull/543

Reply:
@romani done, PR with doc update https://github.com/checkstyle/contribution/pull/544